workflow/rocoto_funcs/jedivar.py

```diff
@@ -26,6 +26,7 @@ def jedivar(xmlFile, expdir, do_spinup=False):
         'REFERENCE_TIME': '@Y-@m-@dT@H:00:00Z',
         'YAML_GEN_METHOD': os.getenv('YAML_GEN_METHOD', '1'),
         'COLDSTART_CYCS_DO_DA': os.getenv('COLDSTART_CYCS_DO_DA', 'true'),
+        'DO_ENVAR_RADAR_REF': os.getenv('DO_ENVAR_RADAR_REF', 'true'),
```

Comment:
Either we set the default value of `DO_ENVAR_RADAR_REF` to `false`
or 
We add `export DO_ENVAR_RADAR_REF=false` to the two rt-jet exp setup files

Reply:
Let set it to false.